py3status/modules/whatismyip.py

```diff
@@ -36,6 +36,9 @@
 except:
     from urllib2 import urlopen
```

Comment:
we now have [self.py3.request()](https://github.com/ultrabug/py3status/blob/master/py3status/py3.py#L873) which returns a [HttpResponse](https://github.com/ultrabug/py3status/blob/master/py3status/request.py#L25)

```
response = self.py3.request(url)
data = response.json()
```